docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2867,20 +2882,14 @@ function removeMember(removeButton) {
 
 function muteMember(muteButton) {
   const meeting = getCurrentMeeting();
-  const unmute = muteButton.getAttribute('data-unmute');
-  const mute = unmute !== 'true';
-
   const participantID = getRadioValue('participant-select');
+  const selectedMember = meeting.members.membersCollection.get(participantID);
+  const muteStatus = selectedMember.isAudioMuted ? 'false' : 'true';
 
   if (meeting) {
-    meeting.mute(participantID, mute).then((res) => {
-      console.log(res, `participant is ${mute ? 'mute' : 'unmute'}`);
-      if (mute) {
-        muteButton.setAttribute('data-unmute', 'true');
-      }
-      else {
-        muteButton.removeAttribute('data-unmute');
-      }
+    meeting.mute(participantID, muteStatus).then((res) => {
+      console.log(res, `participant is ${muteStatus ? 'mute' : 'unmute'}`);
```

Comment:
log should say "muted" or "unmuted"

Reply:
It was an existing log so didn't touch it. Updated it now